index.js

```diff
@@ -11,3 +12,25 @@ mongoose
   })
   .then(x => console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`))
   .catch(err => console.error('Error connecting to mongo', err));
+
+// const recette = { title: 'Bolognaise', level: 'Easy Peasy', ingredients: ['pasta','tomato sauce'], cuisine : 'italian'};
+
+// Recipe.create(recette)
+//   .then(recipe => console.log('The recipe is saved and its value is: ', recipe))
+//   .catch(error =>
+//     console.log('An error happened while saving a new recipe:', error)
+//   );
+
+  // Recipe.insertMany(data, function(error, docs) {});
+
+  // Recipe.updateOne({ title: 'Rigatoni alla Genovese' }, { duration: 100 })
+  // .then()
+  // .catch();
+  
+  Recipe.deleteOne({ title: 'Carrot Cake' })
+  .then(console.log("success, you deleted Carrot Cake !"))
+  .catch();
```

Comment:
I noticed you don't log your errors in the catch block, it would be better if you did as you'd know where / which promise failed along the line.
It doesn't matter for the purpose of this lab but on a bigger project with many lines of code it would be easier to track down errors.